tests/integration/olp-cpp-sdk-authentication/TokenProviderTest.cpp

```diff
@@ -191,4 +190,43 @@ TEST_F(TokenProviderTest, SingleTokenMultipleUsers) {
   testing::Mock::VerifyAndClearExpectations(network_mock_.get());
 }
 
+TEST_F(TokenProviderTest, UseServerTime) {
+  {
+    SCOPED_TRACE("Request token, use server time");
+
+    EXPECT_CALL(*network_mock_, Send(IsGetRequest(kTimestampUrl), _, _, _, _))
+        .WillOnce(ReturnHttpResponse(GetResponse(http::HttpStatusCode::OK),
+                                     kResponseTime));
+
+    EXPECT_CALL(*network_mock_, Send(IsPostRequest(kOAuthTokenUrl), _, _, _, _))
```

Comment:
Can you please also correct this in L159 as I mistakenly market it as GET request.